maquetado/scss/login.scss

```diff
@@ -0,0 +1,68 @@
+.login-container {
+  background-color: $wild-sand;
+  border-top: 8px solid $cerulean;
+  display: flex;
+  flex-flow: column wrap;
+  height: 100%;
+  justify-content: center;
+  max-height: 400px;
+  max-width: 300px;
+  width: 100%;
+  
+  .login-bottom {
+    padding: 20px 24px 24px;
+
+    .wolox-logo {
+      height: 70px;
+      max-height: 100%;
+      max-width: 212.42px;
+      width: 100%;
+    }
+
+    .login-form {
+      border-bottom: 2px solid $grey;
+      display: flex;
+      flex-flow: column wrap;
+      flex-grow: 1;
+      padding-bottom: 16px;
+      width: 100%;
+
+      .login-label {
+        font-size: $font-small;
+        margin-bottom: 8px;
+      } 
+
+      .input-form {
+        background-color: $white;
+        border-radius: 5px;
+        font-size: $font-small;
+        height: 100%;
+        margin-bottom: 16px;
+        max-height: 32px;
+        max-width: 252px;
+        padding: 8px 0;
+        width: 100%;
+      }
+
+      .login-button {
+        background-color: $sulu-green;
+        border-bottom: 2px solid $grey;
+        border-radius: 5px;
+        color: $wild-sand;
+        font-size: $font-small;
+        margin-top: 8px;
+        padding: 8px 0;
+      }
+    }
+
+    .sign-up {
+      border-radius: 5px;
+      border: 2px solid $sulu-green;
+      color: $sulu-green;
+      font-size: $font-small;
+      margin-top: 16px;
```

Comment:
Instead of add margin-top here, add margin-bottom: 16px to `login-form`

Reply:
Done!